app/views/lessons/show.html.erb

```diff
@@ -10,5 +10,53 @@
                 </div>
             </div>
         </div>
+                        <strong>Result: <%= @category.title %></strong>
```

Comment:
![image](https://user-images.githubusercontent.com/35232348/105292667-e4e00800-5bf3-11eb-8cfa-57ab21d8820f.png)
What is this spacing? Improve indentions for this file. Not just this line, but everything in this file, show.html.erb.